spring-cloud-gateway-server-mvc/src/main/java/org/springframework/cloud/gateway/server/mvc/filter/LoadBalancerFilterFunctions.java

```diff
@@ -169,8 +171,7 @@ public int getPort() {
 
 		@Override
 		public boolean isSecure() {
-			// TODO: move to map
-			if ("https".equals(this.overrideScheme) || "wss".equals(this.overrideScheme)) {
+			if (SECURE_SCHEMES.contains(this.overrideScheme)) {
```

Comment:
I don't even think we need to check if the scheme is `wss`, from looking at the code, `overrideScheme` can only be `http` or `https`.